flink-yarn-tests/src/test/java/org/apache/flink/yarn/YARNSessionCapacitySchedulerITCase.java

```diff
@@ -189,9 +189,9 @@ public void perJobYarnClusterOffHeap() throws IOException {
 	/**
 	 * Test TaskManager failure and also if the vcores are set correctly (see issue FLINK-2213).
 	 */
+	@Ignore
 	@Test(timeout = 100000) // timeout after 100 seconds
 	public void testTaskManagerFailure() throws Exception {
-		assumeTrue("The new mode does not start TMs upfront.", !isNewMode);
```

Comment:
@tillrohrmann You mean I should also keep the `assumeTrue` and remove `@Ignore`?  But in other PR, it caused test error : 

```
testClientStartup(org.apache.flink.yarn.YARNSessionCapacitySchedulerITCase)  Time elapsed: 10.063 sec  <<< ERROR!
org.junit.AssumptionViolatedException: The new mode does not start TMs upfront.
	at org.junit.Assume.assumeTrue(Assume.java:59)
	at org.apache.flink.yarn.YARNSessionCapacitySchedulerITCase.testClientStartup(YARNSessionCapacitySchedulerITCase.java:106)
```

Reply:
Where was this the case? I thought if assumptions are violated, then the test will be ignored.

Reply:
@tillrohrmann see also my last comment above. For quick ref, the corresponding log link to https://api.travis-ci.org/v3/job/439612829/log.txt

Reply:
It's here : https://api.travis-ci.org/v3/job/439612829/log.txt . It seems there is other reason, but this error message logged in the log file.

Reply:
I think the problem is that there was another precondition violated which caused a `MultipleFailureException` to be thrown and due to that the `AssumptionException` was not properly treated as ignoring the respective test case.

Reply:
@tillrohrmann So, is your review suggestion still the same? Do we still need to keep the isNewMode field? (About these methods, I have created a [separate issue](https://issues.apache.org/jira/browse/FLINK-10558) to track them, I think we will not forget them in the end).

Reply:
I think it would be cleaner to keep `isNewMode` and remove the `@Ignore` annotations. The issue will still be valid to track porting the remaining test cases. But I'm also not super adamant on reverting the changes.

Reply:
OK, then I will accept this suggestion.